tests/dummy/app/helpers/check-data-type.js

```diff
@@ -0,0 +1,7 @@
+import Ember from 'ember';
+
+export function checkDataType(resource) {
```

Comment:
You're right, that will be better.  My goal for this PR was to try to quickly test the model and make sure we could create, update, delete drafts, and create registrations (testing the new API v2 functionality).   In the future, the forms for these drafts will need to be improved.